docs/content/tutorial/index.ngdoc

```diff
@@ -129,30 +131,32 @@ Once you have Node.js installed on your machine, you can download the tool depen
 npm install
 ```
 
-This command reads angular-phonecat's `package.json` file and downloads the following tools
-into the `node_modules` directory:
+This command reads angular-phonecat's `package.json` file and downloads the following tools into the
+`node_modules` directory:
 
-- [Bower][bower] - client-side code package manager
-- [Http-Server][http-server] - simple local static web server
-- [Karma][karma] - unit test runner
-- [Protractor][protractor] - end to end (E2E) test runner
+* [Bower][bower] - client-side code package manager
+* [Http-Server][http-server] - simple local static web server
+* [Karma][karma] - unit test runner
+* [Protractor][protractor] - end-to-end (E2E) test runner
 
-Running `npm install` will also automatically use bower to download the Angular framework into the
+Running `npm install` will also automatically use bower to download the AngularJS framework into the
```

Comment:
I thought about it, but (afaik) we would need a "build" step that copies the appropriate files from `node_modules/` into `app/`. So, it decided it would be preferrable to keep things simple.